.github/workflows/build.yml

```diff
@@ -23,8 +23,8 @@ jobs:
           - {IMAGE: azure, VERSION: 2}
           - {IMAGE: chrome, VERSION: 1}
           - {IMAGE: dind, VERSION: 1}
-          - {IMAGE: ibm, VERSION: 1.4}
           - {IMAGE: golang, VERSION: 1.14}
+          - {IMAGE: ibm, VERSION: 1.4}
```

Comment:
These changes are already on master. Can you rebase your branch please @Mallear ?

Otherwise, LGTM :+1:

Reply:
Done